src/shell/Shell.cpp

```diff
@@ -813,8 +890,27 @@ int main(int argc, char* argv[])
     Store* store = new Store(engine);
 
     SpecTestFunctionTypes functionTypes;
+    bool runAllExports = false;
+    std::string entry;
 
     for (int i = 1; i < argc; i++) {
+        if (argv[i][0] == '-') {
+            if (strcmp(argv[i], "--run-all-exports") == 0) {
+                runAllExports = true;
+                continue;
+            }
+            if (strcmp(argv[i], "--entry") == 0) {
```

Comment:
I would not use two arguments for this. Maybe:
`--run-export *` runs all exports
`--run-export name` run one export